baselines/iam-service-test/test/gapic_key_management_service_v1.ts.baseline

```diff
@@ -0,0 +1,2657 @@
+                projectId: 'bogus',
+            });
+            client.initialize();
+            const request = generateSampleMessage(new protos.google.cloud.kms.v1.ListImportJobsRequest());
+            request.parent = '';
+            const expectedHeaderRequestParams = "parent=";const expectedResponse = [
+              generateSampleMessage(new protos.google.cloud.kms.v1.ImportJob()),
+              generateSampleMessage(new protos.google.cloud.kms.v1.ImportJob()),
+              generateSampleMessage(new protos.google.cloud.kms.v1.ImportJob()),
+            ];
+            client.descriptors.page.listImportJobs.asyncIterate = stubAsyncIterationCall(expectedResponse);
+            const responses: protos.google.cloud.kms.v1.IImportJob[] = [];
+            const iterable = client.listImportJobsAsync(request);
+            for await (const resource of iterable) {
+                responses.push(resource!);
+            }
+            assert.deepStrictEqual(responses, expectedResponse);
+            assert.deepStrictEqual(
+                (client.descriptors.page.listImportJobs.asyncIterate as SinonStub)
+                    .getCall(0).args[1], request);
+            assert.strictEqual(
+                (client.descriptors.page.listImportJobs.asyncIterate as SinonStub)
+                    .getCall(0).args[2].otherArgs.headers['x-goog-request-params'],
+                expectedHeaderRequestParams
+            );
+        });
+
+        it('uses async iteration with listImportJobs with error', async () => {
+            const client = new keymanagementserviceModule.v1.KeyManagementServiceClient({
+                credentials: {client_email: 'bogus', private_key: 'bogus'},
+                projectId: 'bogus',
+            });
+            client.initialize();
+            const request = generateSampleMessage(new protos.google.cloud.kms.v1.ListImportJobsRequest());
+            request.parent = '';
+            const expectedHeaderRequestParams = "parent=";const expectedError = new Error('expected');
+            client.descriptors.page.listImportJobs.asyncIterate = stubAsyncIterationCall(undefined, expectedError);
+            const iterable = client.listImportJobsAsync(request);
+            await assert.rejects(async () => {
+                const responses: protos.google.cloud.kms.v1.IImportJob[] = [];
+                for await (const resource of iterable) {
+                    responses.push(resource!);
+                }
+            });
+            assert.deepStrictEqual(
+                (client.descriptors.page.listImportJobs.asyncIterate as SinonStub)
+                    .getCall(0).args[1], request);
+            assert.strictEqual(
+                (client.descriptors.page.listImportJobs.asyncIterate as SinonStub)
+                    .getCall(0).args[2].otherArgs.headers['x-goog-request-params'],
+                expectedHeaderRequestParams
+            );
+        });
+    });
+});
```

Comment:
IAM methods should be covered by the generated unit tests (both callback and `await` use cases, both error and no-error cases, similarly to how it's done for regular simple methods).

Reply:
Sure, adding now.

Reply:
done.